e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -120,35 +129,74 @@
               "inputName": {
                 "value": "input2"
               },
+              "testResultCoordinatorUrl": {
+                "value": "http://testResultCoordinator:5001/api/testoperationresult"
+              }
+            },
+            "settings": {
               "image": "<Container_Registry>/microsoft/azureiotedge-relayer:<Build.BuildNumber>-linux-<Architecture>",
               "createOptions": ""
             },
-            "networkController": {
+            "testResultCoordinator": {
               "version": "1.0",
               "type": "docker",
               "status": "running",
               "restartPolicy": "always",
+              "env": {
+                "trackingId": {
```

Comment:
Please add "TestDuration" environment variable

Reply:
TestDuration is based on the frequencies and runscount

---
In reply to: [357393496](https://github.com/Azure/iotedge/pull/2112#discussion_r357393496) [](ancestors = 357393496)